<launch>

    <arg name="config" default="$(find multiverse_client)/config/multiverse_socket.yaml" />

    <node pkg="multiverse_client" type="multiverse_service.py" name="multiverse_service"
        output="screen" args="7400" />

    <node pkg="multiverse_client" type="multiverse_publisher.py" name="multiverse_publisher"
        output="screen">
        <rosparam command="load" file="$(arg config)" />
    </node>

</launch>